src/common/templates/check-your-answers.njk

```diff
@@ -179,51 +179,51 @@
-                    <span class="summary btn-link">{{__('label_manifest_further_personal_details')}}</span>
-                  </summary>
-                    <div class="panel panel-border-narrow" id="details-content-2" aria-hidden="true">
-                      <table class="govuk-table" width="100%">
-                        <tbody class="govuk-table__body">
-                          <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">{{__('field_gender')}}</td>
-                            <td class="govuk-table__cell">{{person.gender}}</td>
-                          </tr>
-                          <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">{{__('field_birth_place')}}</td>
-                            <td class="govuk-table__cell">{{person.placeOfBirth}}</td>
-                          </tr>
-                          <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">{{__('field_travel_document_type')}}</td>
-                            <td class="govuk-table__cell">{{person.documentType}}</td>
-                          </tr>
-                          <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">{{__('field_travel_document_issuing_state')}}</td>
-                            <td class="govuk-table__cell">{{person.issuingState | upper}}</td>
-                          </tr>
-                          <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">{{__('field_travel_document_expiry_date')}}</td>
-                            <td class="govuk-table__cell">{{person.documentExpiryDate}}</td>
-                          </tr>
-                        </tbody>
-                      </table>
-                    </div>
-                </details>
-              </div>
-            </td>
-          </tr>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell">{{person.lastName}}</td>
+              <td class="govuk-table__cell">{{person.firstName}}</td>
+              <td class="govuk-table__cell">{{person.dateOfBirth}}</td>
+              <td class="govuk-table__cell">{{person.documentNumber}}</td>
+              <td class="govuk-table__cell">{{person.nationality | upper}}</td>
+              <td class="govuk-table__cell">{{person.peopleType.name}}</td>
+            </tr>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell" colspan="6">
+                <div>
+                  <details role="group">
+                    <summary role="button" aria-controls="details-content-{{ loop.index }}" aria-expanded="false" title="{{__('further_information_for', person.firstName, person.lastName)}}">
+                      <span id="further-information-{{ loop.index }}" class="summary btn-link">{{__('label_manifest_further_personal_details')}}</span>
```

Comment:
This was previously a convoluted xpath in the E2E test so an id simplifies things